parsers/ocaml.c

```diff
@@ -1809,9 +2024,20 @@ static void localScope (vString * const ident, ocaToken what)
 static void computeModuleName ( void )
 {
 	/* in Ocaml the file name define a module.
-	 * so we define a module =)
+	 * so we define a module if the file has
+	 * things in it. =)
 	 */
 	const char *filename = getSourceFileName ();
+
+	FILE *file = fopen (filename, "r");
+	fseek (file, 0, SEEK_END);
+	long fileSize = ftell (file);
+	fclose (file);	// CLOSE!
+	if (fileSize == 0)
+	{
+		return;
+	}
```

Comment:
It looks like that's the current behavior. ...weird. I was certain I had changed that. But I guess not. I'll update my other comment to reflect this.